AGGREGATE.md

```diff
@@ -587,7 +587,8 @@ contribution to any one bucket). Care should be taken to ensure that either:
 * We can support multiple types of contribution bounding for different reporting
   origins without introducing privacy leaks
 
-See [issue 249](https://github.com/WICG/conversion-measurement-api/issues/249)
+See [issue 249](https://github.com/WICG/attribution-reporting
```

Comment:
This link seems to be broken, we don't need newline here.

Reply:
Fixed, good catch!